meow-command.el

```diff
@@ -1560,10 +1560,26 @@ To search backward, use \\[negative-argument]."
 Before jump, a mark of current location will be created."
   (interactive)
   (meow--cancel-selection)
-  (unless (member last-command '(meow-pop-to-mark-command meow-unpop-to-mark-command))
+  (unless (member last-command '(meow-pop-to-mark-command meow-unpop-to-mark-command meow-pop-or-unpop-to-mark))
```

Comment:
Aren't `meow-pop-to-mark-command` and `meow-unpop-to-mark-command` non-existent commands? I noticed this when I wrote #679, but didn't change it beacuse I thought about making a new PR.

Reply:
Introduced in e2ff708 by @DogLooksGood: did you mean to wrote `meow-pop-to-mark` and `meow-unpop-to-mark`, or was this just a typo?

Reply:
(issue opened for this: https://github.com/meow-edit/meow/issues/697)